src/thunderbird_accounts/subscription/views.py

```diff
@@ -59,28 +63,50 @@ def get_paddle_information(request: Request):
 
 
 @login_required
-@require_http_methods(['POST'])
-@inject_paddle
-def notify_paddle_checkout_complete(request: Request, paddle: Client):
-    data = json.loads(request.body.decode())
-    transaction_id = data.get('transactionId')
+@require_http_methods(['PUT'])
+def set_paddle_transaction_id(request: Request):
+    """This error is used to work-around the fact we don't get a transaction id from Paddle's successUrl redirect.
+    It kinda sucks, but it works for now."""
+    if settings.IS_DEV:
+        data = json.loads(request.body.decode())
+        request.session[SESSION_PADDLE_TRANSACTION_ID] = data.get('txid')
+    return JsonResponse({'success': True})
```

Comment:
This shouldn't get called in non-dev environments but in case it does we just return true.